feature/experimental/p4rt/ate_tests/base_p4rt/base_p4rt_test.go

```diff
@@ -280,16 +280,45 @@ func setupP4RTClient(ctx context.Context, args *testArgs) error {
 	return nil
 }
 
-// Function to compare and check if the expected table is present in RPC ReadResponse
+// Function to compare and check if the expected table is present in RPC ReadResponse.
 func verifyReadReceiveMatch(expected_update []*p4_v1.Update, received_entry *p4_v1.ReadResponse) error {
 
 	matches := 0
 	for _, table := range received_entry.Entities {
-		if diff := cmp.Diff(expected_update[0].Entity.Entity, table.Entity, protocmp.Transform()); diff != "" {
-			glog.Errorf("Table entry diff (-want +got): \n%s", diff)
-			continue
+		// Added a switch case for the captured table entry.
+		// Switch case to remove the counter_data and meter_config from table entry returned by p4rt server.
+		var tableMap map[string]any
+		switch entry := table.Entity.(type) {
+		case *p4_v1.Entity_TableEntry:
+			ent1, err1 := json.Marshal(entry)
+			if err1 != nil {
+				return errors.New("Unable to convert table entry to json")
+			}
+			var toMap map[string]any
+			err2 := json.Unmarshal([]byte(string(ent1)), &toMap)
+			if err2 != nil {
+				return errors.New("Unable to unmarshal table entry to map")
+			}
+			tableMap = toMap["TableEntry"].(map[string]any)
+			delete(tableMap, "meter_config")
+			delete(tableMap, "counter_data")
+		default:
+			fmt.Println("Not a table entry")
```

Comment:
logging shouldn't use fmt.Println. Either just skip these entries or change the function to accept *testing.T and use t.Log

Reply:
Hi Greg,

I will address this in the next commit.

Reply:
The issue has been fixed. Thanks.